lib/aker_permission_client_config.rb

```diff
@@ -0,0 +1,33 @@
+module AkerPermissionClientConfig
+  def has_permission?(user_email, role)
+    mapping = {read: :r, write: :w, execute: :x}    
```

Comment:
I don't think we should hard code the permissions in here. Maybe just let the user pass in the permission they want to check.